lib/cryptoexchange/exchanges/rightbtc/services/market.rb

```diff
@@ -40,12 +40,12 @@ def adapt(output, market_pair)
           ticker.base      = market_pair.base
           ticker.target    = market_pair.target
           ticker.market    = Rightbtc::Market::NAME
-          ticker.ask       = NumericHelper.to_d(output['sell'])
-          ticker.bid       = NumericHelper.to_d(output['buy'])
-          ticker.last      = NumericHelper.to_d(output['last24h'])
-          ticker.high      = NumericHelper.to_d(output['high24h'])
-          ticker.low       = NumericHelper.to_d(output['low24h'])
-          ticker.volume    = NumericHelper.to_d(output['vol24h'].abs)
+          ticker.ask       = NumericHelper.to_d(output['sell'].to_f/(10**8))
```

Comment:
minor change, but how about storing the 10**8 in a variable?

Reply:
sorry, done!!